js/pricing.js

```diff
@@ -28,6 +58,67 @@ $(
                 disabledElement.attr('disabled', true);
                 disabledElement.addClass('disabled');
             }
+        };
+
+        /**
+         * Submit order handler.
+         */
+        const submitOrder = () => {
+            const dataProcessingConsent = $confirmPersonalInformation.prop("checked");
+            const supportAgreementConsent = $confirmDevelopmentAgreement.prop("checked");
+            const data = JSON.stringify({
+                "dataProcessingConsent": dataProcessingConsent,
+                "supportAgreementConsent": supportAgreementConsent
+            });
```

Comment:
You can avoid param names when the value name is the same. This should work as well:

```suggestion
            const data = JSON.stringify({dataProcessingConsent, supportAgreementConsent});
```